decidim-accountability/app/views/decidim/accountability/results/_timeline.html.erb

```diff
@@ -13,8 +13,18 @@
                 <%= timeline_entry.entry_date %>
               </span>
               <h4 class="timeline__title">
-                <%= translated_attribute timeline_entry.description %>
+                <%= translated_attribute timeline_entry.title %>
               </h4>
+
+              <% if translated_attribute(timeline_entry.description).present? %>
+                <a data-toggle="timeline_entry__description_toggle_<%= timeline_entry.id %>" class="button small hollow">
+                  <%= t(".read_more") %>
```

Comment:
There are some accessibility issues introduced from this button:
1. This should not be a link element (`<a>`) because it's not linking anywhere. It should be a `<button>` instead.
2. When I load the page and have not clicked on the button yet, the element has an `aria-expanded="true"` attribute which indicates the item is open when it's not actually open. It should be `aria-expanded="false"` when it is closed and `aria-expanded="true"` when it is open.
3. When the item is opened, the button should state "Show less" instead of "Read more". Maybe we should also apply the same name for the closed state, so "Show more" and "Show less"?
4. When we have multiple buttons that do the same thing on the page, accessibility wise we should always indicate what they are doing. Many people using assistive technologies will quickly glance through the page with all its interactive elements by using the `TAB` key. Now when they browse the page with multiple of these elements, the screen reader would just read many times "Read more" consecutively. Therefore, we should always state explicitly what THIS button is doing. Read more **about what?**

Reply:
@agustibr It was concluded at #9089 that we should remove the "Read more" toggle buttons, so let's do that.